src/services/qix/index.js

```diff
@@ -257,6 +259,9 @@ export default class Qix {
         config.session.host = 'localhost';
       }
     }
+    if (typeof config.secure === 'undefined') {
```

Comment:
My bad, the github PR context showed `buildUrl`, then minimized to your code. :)